app/models/header.js

```diff
@@ -0,0 +1,40 @@
+"use strict";
+
+const Header = function (data, db) {
+  this.data = data;
+  this.db = db;
+};
+
+Header.prototype.get = (name) => {
```

Comment:
@stulees I noticed that as well when testing the function. Fixed that on the next commit. cheers